### PR TITLE
docs/migrate to docusaurus

### PR DESCRIPTION
**Pull request by aleksei-korolev**

### CODE REVIEW

File: docOps/blog/2019-05-28-hola.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (lucas-tortora) [inline docOps/blog/2019-05-28-hola.md:1]: We can delete the entire blog folder for the time being.

File: docOps/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'Chronicle',
+  tagline: 'Official IOTA permanode solution.',
+  url: 'https://docs.iota.org', // TODO: PROPER URL
+  baseUrl: '/chronicle.rs/',
+  onBrokenLinks: 'warn',
+  onBrokenMarkdownLinks: 'throw',
+  favicon: '/img/logo/img/favicon.ico',
```
Comment (lucas-tortora) [inline docOps/docusaurus.config.js:12]: favicon is missing.
```suggestion
  favicon: '/img/logo/img/favicon.ico',
```

File: docOps/src/components/HomepageFeatures.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```
Comment (lucas-tortora) [inline docOps/src/components/HomepageFeatures.js:1]: We can delete this file, it's the default docusaurus home page.

File: docOps/src/components/HomepageFeatures.module.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/* stylelint-disable docusaurus/copyright-header */
```
Comment (lucas-tortora) [inline docOps/src/components/HomepageFeatures.module.css:1]: We can delete this file, it's the default docusaurus home page css.

**Review by lucas-tortora — COMMENTED** [review]
A few files which can be delete and a minor config touch up

**Comment by aleksei-korolev** [comment]
@lucas-tortora done

**Review by lucas-tortora — APPROVED** [review]

**Review by bingyanglin — APPROVED** [review]
LGTM. Also i guess we can add the API documentation and add more examples as Jack did last year. Thanks!

**Pull request merged**